include/cjdb/contracts.hpp

```diff
@@ -24,7 +31,21 @@
 	#define CJDB_PRETTY_FUNCTION __PRETTY_FUNCTION__
 #endif // _MSC_VER
 
-namespace cjdb::contracts_detail {
+namespace cjdb {
+	using print_error_fn = void(std::string_view);
+	inline print_error_fn* print_error = [](std::string_view message) {
+#ifdef CJDB_USE_IOSTREAM
+		std::cerr.write(message.data(), static_cast<std::streamsize>(message.size()));
+#elif !defined(CJDB_SKIP_STDIO)
```

Comment:
What is the motivation for skipping I/O?

Reply:
1. For use in freestanding implementations that lack an output device and/or the header `cstdio`
2. For users who provide their own `cjdb::print_error` function. While this use case is possible without a configuration option, it's not necessarily desirable to `#include <cstdio>` with all that implies (e.g. declaring a function called `remove` in the global namespace).